components/ad/blockerExtensionDetector/src/index.js

```diff
@@ -0,0 +1,48 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+
+class AdBlockerExtensionDetector extends Component {
+  constructor(props) {
+    super(props)
+    this.blockDetector = React.createRef()
+    this.state = {
+      displayBanner: false
+    }
+  }
+
+  _detectIfAdBlockerIsEnabled = () => {
+    const blockDetector = this.blockDetector.current
+
+    const isBlockerAdviseActive = blockDetector.offsetHeight === 0
+    this.setState(
+      {
+        displayBanner: isBlockerAdviseActive
+      },
+      () => blockDetector.remove()
+    )
+  }
+
+  componentDidMount() {
+    this._detectIfAdBlockerIsEnabled()
+  }
+
+  render() {
+    const {children} = this.props
+    return (
+      <React.Fragment>
+        {this.state.displayBanner && <div>{children}</div>}
```

Comment:
Is it really necessary to wrap the children inside a div?

Reply:
You are right! Isn't necessary! Thanks Julian I will fixe it 👍